trunner/testcase.py

```diff
@@ -39,10 +39,14 @@ def __init__(
         self.status = status
         self.harness = None
         self.exception = ''
+        self.type = None
 
     def fail(self):
         self.status = TestCase.FAILED
 
+    def skip(self):
```

Comment:
deciding which test to run should be done only by inspecting YAML files.

Reply:
solved with `is_type` logic removed